modules/nf-core/rtgtools/rocplot/tests/main.nf.test

```diff
@@ -0,0 +1,69 @@
+nextflow_process {
+
+    name "Test Process RTGTOOLS_ROCPLOT"
+    script "../main.nf"
+    process "RTGTOOLS_ROCPLOT"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "rtgtools"
+    tag "rtgtools/rocplot"
+
+    test("homo_sapiens - tsv") {
+
+        setup {
+            run("UNTAR") {
+                script "../../../untar/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [ id:'test' ],
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_sdf'])
+                    ]
+                    """
+                }
+            }
+
+            run("RTGTOOLS_VCFEVAL") {
+                script "../../../rtgtools/vcfeval/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [ id:'test' ], // meta map
+                        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_vcf_gz'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_vcf_gz_tbi'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_ann_vcf_gz'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_ann_vcf_gz_tbi'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['genome']['genome_bed'], checkIfExists: true)
+                    ]
+                    input[1] = UNTAR.out.untar
+                    """
+                }
+            }
+        }
+
+        when {
+            params {
+                outdir = $outputDir
+            }
+            process {
+                """
+                input[0] = RTGTOOLS_VCFEVAL.out.weighted_roc
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.png[0][1]).exists(),
+                    file(process.out.svg[0][1]).exists(),
+                    file(process.out.versions[0]).exists()
```

Comment:
Are these files md5sum-able? If yes, you should remove `exists()`, otherwise, you should probably use `assert file(process.out.png[0][1]).exists()`.

Reply:
Nope the md5sums differ everytime, that's why I added these checks :p

Reply:
I've added the file name as a snapshot value, I think this is better than to dynamically check if the output exists (without checking the filename). this way we know for sure the right files are in the right channel